{{cookiecutter.app_name}}/README.rst

```diff
@@ -50,32 +47,30 @@ Shell
 
 To open the interactive shell, run ::
 
-    python manage.py shell
+    flask shell
 
-By default, you will have access to ``app``, ``db``, and the ``User`` model.
+By default, you will have access to ``app``, ``db``, ``g``, and the ``User`` model.
```

Comment:
This isn't quite accurate. The default `flask shell` command only gives you access to the `app`.